### ISSUE TITLE
Improve RDT documentation

### ISSUE THREAD

**Issue by msivosuo** [intel/cri-resource-manager#320]
Update RDT documentation, better describing partitions, classes, configuration, exclusivity etc.

**Comment by msivosuo** [comment]
@marquiz - is this covered by https://github.com/intel/cri-resource-manager/pull/426?

**Comment by marquiz** [comment]
> @marquiz - is this covered by #426?

No, if I'm being honest 😄 There's still improvements to do. I'd like to cover the details a bit better.

**Comment by msivosuo** [comment]
ok:)
